src/main/java/projectbookstore/dto/book/CreateBookRequestDto.java

```diff
@@ -28,4 +30,6 @@ public class CreateBookRequestDto {
     private String description;
     @NotBlank
     private String coverImage;
```

Comment:
coverimage and description can be empty according to task . So we can remove @NotBlank